### DISCUSSION TITLE
Switching to version 5.4.0 broke the system producing a 500 error

### DISCUSSION THREAD

**Discussion by bbmmbb** [snipe/snipe-it#10743] [General]
Switching to version 5.4.0 broke the system producing a 500 error. There was no output to storage/logs/laravel.log indicating what the problem might be. I reverted to 5.3.10 with:

sudo git checkout tags/5.3.10 -b master

**Comment by rslygh** [comment]
I ran across your post here trying to resolve a 500 issue of my own, and mine turned out to be a combo of the .htaccess file and my Apache config. I don't imagine it's the same as what you're experiencing since you were able to resolve by checking out a prior version, and my issue should persist through checkouts, but I'll leave my situation in case it's helpful to you or anyone else that comes across your question. 

The person who had installed Snipe-IT in our environment had been removing the .htaccess file in the root directory after checkout of a new version and hadn't told anyone. If you didn't remove this .htaccess after checking out a new version, the site was throwing a 500 error.

The real issue was that Apache wasn't set to allow overrides by default, and the default [Snipe-IT Apache documentation](https://snipe-it.readme.io/docs/linuxosx) only lists your snipeit/public path for allowing overrides. However, if you don't allow overrides in the root directory, or by default within Apache, the .htaccess in the root Snipe-IT will cause a 500 error.

To fix, I went into my Snipe-IT site's Virtualhost file and added in 

`    
<Directory /var/www/html/snipeit>
AllowOverride All
</Directory>
`

After that the site loaded fine and no more 500 error. Just make sure you're using the path to your Snipe-IT install, which may differ from what I have there.

**Comment by aranar-pro** [comment]
@bbmmbb Just to confirm, did you try running any of the following commands manually? https://snipe-it.readme.io/docs/upgrading#after-pulling-the-new-version-files ?  There's a bunch here too that can be really helpful: https://snipe-it.readme.io/docs/common-issues#quick-tip-handy-commands. On rare occasion I've also found I've had to run composer update and recycle my application pool in IIS.
